conf/drupal/config/views.view.content.yml

```diff
@@ -4121,14 +4183,12 @@ display:
         type: input_required_on_request
         options:
           submit_button: Apply
-          reset_button: 1
+          reset_button: true
           reset_button_label: Reset
           exposed_sorts_label: 'Sort by'
-          expose_sort_order: 1
+          expose_sort_order: true
           sort_asc_label: Asc
           sort_desc_label: Desc
-          text_input_required: 'Select any filter and click on Apply to see results'
-          text_input_required_format: basic_html
```

Comment:
This part doesnt seem relevant as well @beto-aveiga

Reply:
You are right @arthurbaghdas. Thanks again!